merlin/dataloader/loader_base.py

```diff
@@ -626,9 +626,14 @@ def _handle_tensors(self, tensors, tensor_names):
         # TODO: use dict for labels as well?
         # would require output layers to match naming
         # labels should not exist separately they should be a regular column
-        labels = None
-        if len(self.label_names) > 0:
```

Comment:
So this looks like it was previously not working for multiple columns. (we're silently ignoring targets and keeping them in the input data?)

Do we have any tests or functionality in Transformer4Rec that makes use of multiple targets? (cc. @sararb ) And if so, I'm wondering why this didn't cause any test to fail in this PR https://github.com/NVIDIA-Merlin/Transformers4Rec/pull/547 

I'd like to check that if we have muliple targets in Transformers4Rec and in PyTorch Models in general, that having the output of the 'y' here as a dict is going to work and the preferred option. (Which might inform whether or not we should make this output type configurable at this  stage.)

Reply:
thanks for the explanation @sararb . So this PR looks like it will satisfy both Merlin Models and Transformers4Rec then with the three possible output types as you described in the last comment.